.github/workflows/llvm-tests.yml

```diff
@@ -12,7 +12,7 @@ on:
       - 'llvm/**'
       - '.github/workflows/llvm-tests.yml'
       - '.github/workflows/llvm-project-tests.yml'
-  pull_request:
+  pull_request_target:
```

Comment:
Are you sure you want to change this to pull request target? That will cause the workflow to just build the `main` branch (or whatever the target branch is) rather than the PR branch.

Reply:
No, I was just trying to see if that would fix the error.  I've reverted that change now.

Reply:
Ah. Didn't realize that. Sorry for the noise.